src/modules/api/user/user.controller.ts

```diff
@@ -1,7 +1,24 @@
-import { Controller } from '@nestjs/common';
+import { Controller, Post, Body } from '@nestjs/common';
 import { UserService } from './user.service';
+import { User } from 'src/schemas/user.schema';
 
-@Controller('users/:id')
+@Controller('users')
 export class UserController {
   constructor(private userService: UserService) {}
+
+  @Post()
+  async createUser(@Body() user: User) {
+    user.joinedAt = new Date();
+    const newUser = await this.userService.createUser(user);
+    if (newUser) {
+      return {
+        status: 'OK',
```

Comment:
I think you should return the generic interface response right?

Reply:
+1